src/containers/Root.js

```diff
@@ -11,16 +13,21 @@ export default class Root extends Component {
 
     store.dispatch(
       login({
-        email: 'Root',
-        password: 'Root'
+        email: 'graeme@gmail.com',
+        password: 'password'
       })
     );
+
+    store.dispatch(getTrans());
+    store.dispatch(getTrips());
+
+    setTimeout(() => {
+      store.dispatch(addTran({trip_id: 86, dscript: "biggy"}));
+    }, 10000);
+
   }
 
   render() {
-    return (
-      <Provider store={store}>
-      </Provider>
-    );
+    return <Provider store={store} />;
```

Comment:
Is this file just used for testing purposes?

Reply:
Yeah, shouldn't have pushed this with my Redux changes. My bad  😬

Reply:
Cool! 👍